data/scripts/actions/others/muck_remover.lua

```diff
@@ -29,6 +29,7 @@ function muckRemover.onUse(player, item, fromPosition, target, toPosition, isHot
 			player:addAchievementProgress("Goo Goo Dancer", 100)
 			target:getPosition():sendMagicEffect(CONST_ME_GREEN_RINGS)
 			target:remove(1)
+			player:sendSupplyUsed(item)
```

Comment:
did you forgot to remove these?

Reply:
muck remover is alternative(work on diffrent item) to rusty remover, i thing it should stay